model.js

```diff
@@ -39,4 +39,15 @@ exports.updateArticles = (article_id, body) => {
             return rows[0]
         }
     })
+}
+
+exports.selectComments = (article_id) => {
+    return db.query(`SELECT * FROM comments WHERE article_id=$1 ORDER BY created_at DESC`, [article_id])
+    .then(({rows}) => {
+        if (rows.length === 0){
```

Comment:
This will need some work as if the article exists but has no comments, we would expect an empty array to be returned